sync/build.gradle

```diff
@@ -17,21 +17,17 @@ dependencies {
     testImplementation testFixtures(project('::networking:eth2'))
     testImplementation testFixtures(project(':util'))
 
-    testImplementation 'org.mockito:mockito-core'
-
     integrationTestImplementation testFixtures(project(':bls'))
     integrationTestImplementation testFixtures(project(':ethereum:statetransition'))
     integrationTestImplementation testFixtures(project('::networking:eth2'))
     integrationTestImplementation testFixtures(project(':util'))
 
-    integrationTestImplementation 'org.mockito:mockito-core'
-
     testFixturesImplementation project(':services:serviceutils')
     testFixturesImplementation project(':util')
 
     testFixturesImplementation testFixtures(project(':ethereum:statetransition'))
     testFixturesImplementation testFixtures(project('::networking:eth2'))
     testFixturesImplementation testFixtures(project('::networking:p2p'))
 
-    testFixturesImplementation 'com.google.guava:guava'
+    testFixturesApi 'com.google.guava:guava'
```

Comment:
What is the difference between `testFixturesImplementation` and `testFixturesApi`?

Reply:
As I understand things, this is the same as the difference between `implementation` and `api` ...which is to say, with `api`:
Functionally:  every project that has this project as a dependency, automatically receives that dependency on it's implementation path.
Design wise: whenever a class from a dependency is in the signature of a public method of this project (a library method, to be used by others), that dependency should be an `api`, as any project using this as a dependency is going to also need that class/dependency (at least at runtime, if not also compile time) 

Does that make sense?